packages/x-components/src/utils/options-api.ts

```diff
@@ -0,0 +1,3 @@
+import { DefineComponent, PropType } from 'vue';
```

Comment:
As I said in the other PR, maybe it should be in the types folder and rename it to `AnimationProp`